randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2967,8 +2975,12 @@ Extra - asset_id: collision_camera_036
       Trivial
   > Upper-right Corner
       Grapple Beam or Spider Magnet or Space Jump or Speed Booster or Walljump (Beginner) or Simple IBJ
+  > Door to Central Unit
+      All of the following:
+          After Cataris - breakablemag008
+          Morph Ball or Can Slide
```

Comment:
Since it's trivial to reach upper-right corner (which was not yet changed on this commit), this connection is redundant